rest/src/test/resources/workload_rest

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2016 Yahoo! Inc. All rights reserved.                                                                                                                             

```

Comment:
Is it necessary to have this copy of the file? The copy in workloads/ will be distributed.